src/components/organisms/ChatDrawer/ChatDrawer.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useContext, useEffect, useState } from "react";
+import MessageList from "components/molecules/MessageList";
+import CallOutMessageForm from "components/molecules/CallOutMessageForm";
+import { useForm } from "react-hook-form";
+import { ChatContext } from "components/context/ChatContext";
+import "./ChatDrawer.scss";
+import { useUser } from "hooks/useUser";
+import { useSelector } from "hooks/useSelector";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faCommentDots,
+  faAngleDoubleLeft,
+} from "@fortawesome/free-solid-svg-icons";
+
+interface ChatOutDataType {
+  messageToTheBand: string;
+}
+
+const ChatDrawer = () => {
+  const roomName = "jazz";
+  const { user } = useUser();
+  const { usersById, chats } = useSelector((state) => ({
+    usersById: state.firestore.data.users,
+    chats: state.firestore.ordered.venueChats,
+  }));
+
+  const [isMessageToTheBarSent, setIsMessageToTheBarSent] = useState(false);
+  const [isChatDrawerExpanded, setIsChatDrawerExpanded] = useState(false);
+
+  useEffect(() => {
+    if (isMessageToTheBarSent) {
+      setTimeout(() => {
+        setIsMessageToTheBarSent(false);
+      }, 2000);
+    }
+  }, [isMessageToTheBarSent, setIsMessageToTheBarSent]);
+
+  const chatContext = useContext(ChatContext);
+  const onBarMessageSubmit = async (data: ChatOutDataType) => {
+    chatContext &&
+      user &&
+      chatContext.sendRoomChat(user.uid, roomName, data.messageToTheBand);
+    setBarMessageValue([{ messageToTheBand: "" }]);
```

Comment:
did this not cause an error using it before it is defined? Place the function below useForm

Reply:
also, an alternative would be to reset the form on submit: https://react-hook-form.com/api/#reset